orca/producer_test.go

```diff
@@ -255,10 +258,14 @@ func (s) TestProducerBackoff(t *testing.T) {
 
 	// Provide a convenient way to expect backoff calls and return a minimal
 	// value.
-	expectedBackoff := -1 // -1 indicates any value is allowed.
+	const backoffShouldNotBeCalled = 9999 // Use to assert backoff function is not called.
```

Comment:
Can you clarify why?

Reply:
I do at the usage points: https://github.com/grpc/grpc-go/pull/5669/files#diff-6a71af69cb3159cbd1bbb20fb808da29d460ee437f134b485afc11df20079bb3R343-R345

Reply:
I don't get that explanation, but it's probably just me.